### PR TITLE
Added UpdateSinkCommand

### PR DESCRIPTION
**Pull request by tmatsuo**

### CODE REVIEW

File: logging/test/UpdateSinkCommandTest.php [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+        $commandTester->execute(
+            [
+                '--project' => $this->projectId,
+                '--sink' => self::$sinkName,
+            ],
+            ['interactive' => false]
+        );
+    }
+
+    public function testListSinks()
```
Comment (bshaffer) [inline logging/test/UpdateSinkCommandTest.php]: Can we add ` testListSinksWithFilter`?

Comment (tmatsuo) [inline logging/test/UpdateSinkCommandTest.php]: Ah sorry this function should be `testUpdateSink`.

Comment (tmatsuo) [inline logging/test/UpdateSinkCommandTest.php]: @bshaffer 
Added the test with filter. PTAL

**Comment by bshaffer** [comment]
LGTM!!

File: logging/src/UpdateSinkCommand.php [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+                'The name of the Logging sink',
+                'my_sink'
+            )->addOption(
+                'filter',
+                null,
+                InputOption::VALUE_OPTIONAL,
+                'The filter expression for the sink',
+                ''
+            );
+        $this->addCommonOptions();
```
Comment (bshaffer) [inline logging/src/UpdateSinkCommand.php]: Testing starting a review

File: logging/test/UpdateSinkCommandTest.php [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Samples\Logging\Tests;
+
+use Google\Cloud\Logging\LoggingClient;
```
Comment (bshaffer) [inline logging/test/UpdateSinkCommandTest.php]: Testing another review comment

**Pull request merged**